src/processor/result/base_hash_table.cpp

```diff
@@ -34,21 +34,33 @@ void BaseHashTable::computeAndCombineVecHash(const std::vector<ValueVector*>& un
             std::make_unique<ValueVector>(LogicalTypeID::INT64, &memoryManager);
         auto tmpHashCombineResultVector =
             std::make_unique<ValueVector>(LogicalTypeID::INT64, &memoryManager);
-        VectorHashFunction::computeHash(keyVector, tmpHashResultVector.get());
-        VectorHashFunction::combineHash(hashVector.get(), tmpHashResultVector.get(),
-            tmpHashCombineResultVector.get());
+        tmpHashResultVector->state = keyVector->state;
```

Comment:
Do u need to set state here? I think you can just do
```
VectorHashFunction::computeHash(*keyVector, keyVector->state->getSelVector(),
            *tmpHashResultVector, keyVector->state->getSelVector());
```
and similar for tmpHashCombineResultVector